controllers/informationCenter/FileInformationCenterHandler.inc.php

```diff
@@ -133,7 +126,9 @@ function listPastNotes($args, $request) {
 		$templateMgr->assign('currentUserId', $user->getId());
 
 		$templateMgr->assign('notesListId', 'pastNotesList');
-		return $templateMgr->fetchJson('controllers/informationCenter/notesList.tpl');
+		$templateMgr->assign('notesDeletable', false);
```

Comment:
Feel free to use...

    $templateMgr->assign(array(
        'a' => $a,
        'b' => $b,
        ...
    ));

...when there are several params being assigned in a row.